main.go

```diff
@@ -3,64 +3,37 @@ package main
 import (
 	"context"
 	"net/http"
-	"os"
-	"strconv"
 	"time"
 
 	"github.com/NickGowdy/deveui-cli/client"
 	"github.com/NickGowdy/deveui-cli/processor"
-	"github.com/joho/godotenv"
+	"github.com/kelseyhightower/envconfig"
 )
 
-/*
```

Comment:
I felt this comment was too much in a main file. Now there is a makefile that is self-explanatory, but if that is not enough then better to put any additional info in the readme.